script/input/1/mainnet-staging.json

```diff
@@ -1,14 +1,27 @@
 {
-    "ilk": "ALLOCATOR-SPARK-A",
-    "usdcUnitSize": 10,
-    "usdsUnitSize": 10,
+    "admin": "0xd1236a6A111879d9862f8374BA15344b6B233Fbd",
+    "allocatorBuffer": "0x21fd5b6f0FbF7300a23F9cC397630A27Ee013AE5",
+    "allocatorOracle": "0xDf9EE9Ad3cB7ec26F98103751F7Fb7149a284541",
+    "allocatorRegistry": "0x2143B2949bc3B9b405cFebb065aB6beF4DB85956",
+    "allocatorRoles": "0x041918Ef264214BD999776667a693b06B35c36Fa",
+    "allocatorVault": "0x85cb1558802D32D437B63d7C3eB4e6d6c88a383B",
+    "almProxy": "0xba8b8375Af6bB5B7e0424D8581b82fe39CfCff8A",
     "cctpTokenMessenger": "0xBd3fa81B58Ba92a82136038B25aDec7066af3155",
+    "controller": "0xe08828c1A20dA7874A52dA070480B9B1e4213B6C",
     "dai": "0x6B175474E89094C44Da98b954EedeAC495271d0F",
     "daiUsds": "0x3225737a9Bbb6473CB4a45b7244ACa2BeFdB276A",
+    "freezer": "0x611C7c37F296240c2fF5a92f0B4a398B01B237c4",
+    "ilk": "ALLOCATOR-SPARK-A",
+    "jug": "0x6C20DC38D8e978955B12217D82692d448239C005",
     "psm": "0xf6e72Db5454dd049d0788e411b06CfAF16853042",
+    "psmWrapper": "0x66c044CeE6bdf8360C2F6eDffc712829900100fB",
```

Comment:
@krzkaczor this is the wrapper over the PSM that you guys call into. `psm` in this file is the live PSM. I didn't know whether to call this `psm` and the live PSM `livePsm` or the current state, please let me know whats best.

Reply:
I think this is fine 👍 

CC: @nezouse